### ISSUE TITLE
Missing batch_size ?

### ISSUE THREAD

**Issue by MikeZhu92** [udacity/CarND-Keras-Lab#4]
In the solution, 
> Train the Network
1. Compile the network using adam optimizer and categorical_crossentropy loss function.
2. Train the network for ten epochs and validate with 20% of the training data. 

`history = model.fit(X_normalized, y_one_hot, nb_epoch=10, validation_split=0.2)`

Does it missing batch_size = 128 ? I see the batch_size in the similar code in the rest part of the solution. 

Or what will be the default value if we don't assign batch size.
